bin/generate_commit

```diff
@@ -104,6 +104,18 @@ function _create_changelog() {
         --date=iso
 }
 
+function _create_readme(){
+    
+    if [ -e README.md ]; then
+        echo "File README.md already exists!"
+    else
+        echo "iam here"
```

Comment:
You can use a [heredoc](https://tldp.org/LDP/abs/html/here-docs.html) instead of multiple echo lines:

```bash
cat << EOF > README.md
# mozilla-pipeline-schemas generated schemas
This is a description of the branch....
EOF
```